maya/core.py

```diff
@@ -166,6 +166,14 @@ def __dt_to_epoch(dt):
         epoch_start = Datetime(*_EPOCH_START, tzinfo=pytz.timezone('UTC'))
         return (dt - epoch_start).total_seconds()
 
+    @property
+    def aoe(self):
+        """
+        Return True if the date has not passed anywhere on earth.
+        See also: http://www.ieee802.org/16/aoe.html
+        """
+        return when('tomorrow noon') > self > when('yesterday noon')
```

Comment:
Should one of these be inclusive? Python likes to make the lower bound inclusive, so `>= yesterday noon`.